src/Types.h

```diff
@@ -346,7 +346,7 @@ constexpr int from_to(Move m) {
  return m & 0xFFF;
 }
 
-constexpr MoveType type_of(Move m) {
+constexpr MoveType move_type(Move m) {
```

Comment:
Also, this comes from SF - unless there is a reallllly strong reason, I'd rather leave it the same.